assessments/FastBridge_English/seeds/mapping_grade.csv

```diff
@@ -0,0 +1,14 @@
+from,to
+Kindergarten,0
```

Comment:
The default grades should be edfi defaults, and then have those overwritten at run time since SC uses different values than most who may use this bundle